spring-boot-examples/producer-app/src/test/java/io/dapr/springboot/examples/producer/DaprTestContainersConfig.java

```diff
@@ -0,0 +1,121 @@
+package io.dapr.springboot.examples.producer;
+
+import io.dapr.testcontainers.Component;
+import io.dapr.testcontainers.DaprContainer;
+import io.dapr.testcontainers.Subscription;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.context.annotation.Bean;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.RabbitMQContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@TestConfiguration(proxyBeanMethods = false)
+public class DaprTestContainersConfig {
+
+   static final String CONNECTION_STRING =
+          "host=postgres user=postgres password=password port=5432 connect_timeout=10 database=dapr_db_repository";
+   static final Map<String, String> STATE_STORE_PROPERTIES = createStateStoreProperties();
+
+   static final Map<String, String> BINDING_PROPERTIES = Collections.singletonMap("connectionString", CONNECTION_STRING);
+
+
+   @Bean
+   public Network getNetwork() {
+      Network defaultDaprNetwork = new Network() {
+        @Override
+        public String getId() {
+          return "dapr-network";
+        }
+
+        @Override
+        public void close() {
+
+        }
+
+        @Override
+        public Statement apply(Statement base, Description description) {
+          return null;
+        }
+      };
+
```

Comment:
Similar question why we need to use DockerClient

Reply:
check previous answer.